### PR TITLE
2897436 - add support for customization of adaptive card 

### PR DESCRIPTION
**Pull request by elopezanaya**
# Description

Adaptive card are not presented correctly, by default the chat color if assigned to the background of the card .

# Solution
- Add style css support to override the style from adaptive cards
- default values for background is white, and for the font color is black
- expose a property object to allow customization of the same
example 
`                webChatContainerProps: {
                    adaptiveCardStyle: {
                        color: "yellow",
                        background: "orange"
                    }
                }`

# Evidence

## Override background color and font color
image

## override of just font color, background obtained from default
image

## override background , font color obtained from default
image

## Default value 
image

### CODE REVIEW

**Review by dishakhattri — COMMENTED** [review]
Update ChangeLog.md

**Review by sarojkpr — APPROVED** [review]

File: chat-widget/src/components/webchatcontainerstateful/common/defaultProps/defaultWebChatContainerStatefulProps.ts [hunk 1]
```diff
@@ -3,12 +3,14 @@ import MockAdapter from "../mockadapter";
...
 
 export const defaultWebChatContainerStatefulProps: IWebChatContainerStatefulProps = {
     webChatStyles: defaultWebChatStatefulStyles,
     webChatProps: defaultWebChatStatefulProps,
     containerStyles: defaultWebChatStatefulContainerStyles,
     disableNewLineMarkdownSupport: false,
     disableMarkdownMessageFormatting: false,
-    directLine: new MockAdapter()
+    directLine: new MockAdapter(),
+    adaptiveCardStyle : defaultAdaptiveCardStyle
```
Comment (charliewang95) [inline chat-widget/src/components/webchatcontainerstateful/common/defaultProps/defaultWebChatContainerStatefulProps.ts:15]: nit: adaptiveCardStyles[s]

File: chat-widget/src/components/webchatcontainerstateful/common/defaultStyles/defaultAdaptiveCardStyle.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { IAdaptiveCardStyle } from "../../interfaces/IAdaptiveCardStyle";
+
+export const defaultAdaptiveCardStyle : IAdaptiveCardStyle = {
+    background : "white",
```
Comment (charliewang95) [inline chat-widget/src/components/webchatcontainerstateful/common/defaultStyles/defaultAdaptiveCardStyle.ts:4]: nit: remove space

File: chat-widget/src/components/webchatcontainerstateful/common/defaultStyles/defaultAdaptiveCardStyle.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+    color : "black"
```
Comment (charliewang95) [inline chat-widget/src/components/webchatcontainerstateful/common/defaultStyles/defaultAdaptiveCardStyle.ts:5]: nit: correct spacing

File: chat-widget/src/components/webchatcontainerstateful/common/defaultStyles/defaultAdaptiveCardStyle.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import { IAdaptiveCardStyle } from "../../interfaces/IAdaptiveCardStyle";
+
+export const defaultAdaptiveCardStyle : IAdaptiveCardStyle = {
```
Comment (charliewang95) [inline chat-widget/src/components/webchatcontainerstateful/common/defaultStyles/defaultAdaptiveCardStyle.ts:3]: Please update dummyDefaultProps.ts as well

File: chat-widget/src/components/webchatcontainerstateful/common/defaultProps/defaultWebChatContainerStatefulProps.ts (hunk 1 above)
Comment (charliewang95) [inline chat-widget/src/components/webchatcontainerstateful/common/defaultProps/defaultWebChatContainerStatefulProps.ts:15]: nit: spacing around ":"

Comment (elopezanaya) [reply chat-widget/src/components/webchatcontainerstateful/common/defaultProps/defaultWebChatContainerStatefulProps.ts:15]: done

Comment (elopezanaya) [reply chat-widget/src/components/webchatcontainerstateful/common/defaultProps/defaultWebChatContainerStatefulProps.ts:15]: done

File: chat-widget/src/components/webchatcontainerstateful/common/defaultStyles/defaultAdaptiveCardStyle.ts (hunk 4 above)
Comment (elopezanaya) [reply chat-widget/src/components/webchatcontainerstateful/common/defaultStyles/defaultAdaptiveCardStyle.ts:3]: done

**Pull request merged**